DatadogInternal/Sources/FeatureAPIs/CrashReporting/DDCrashReport.swift

```diff
@@ -0,0 +1,102 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-Present Datadog, Inc.
+ */
+
+import Foundation
+
+/// Crash Report format supported by Datadog SDK.
+@objc
```

Comment:
why do we need this to be inter-operable with objc?

Reply:
Fair call, I traced this back to this commit https://github.com/DataDog/dd-sdk-ios/commit/2f892c267a7a4ab4892b4ced2a31aae3b3074dcc where we were adding Obj-c compatibility for Crash Reporting. This was back in V1. With V2 aligning module dependencies much better, this can be now internal to Swift domain, so it is not required. Changed here and consequently in few other places. ✅